insights/parsers/odbc.py

```diff
@@ -0,0 +1,88 @@
+"""
+obdc configuration file - Files
+===============================
+
+Shared mappers for parsing and extracting data from ``/etc/odbc.ini`` and
+``/etc/odbcinst.ini`` files. Parsers contained in this module are:
+
+ODBCIni - file ``/etc/odbc.ini``
+--------------------------------
+
+ODBCinstIni - file ``/etc/odbcinst.ini``
+----------------------------------------
+
+"""
+
+from .. import parser, IniConfigFile, add_filter
+from insights.specs import Specs
+
+filter_list = [
+        '[',
+        'driver',
+        'server',
+        'no_sspa',
```

Comment:
As your test data for ODBCIni, there are "Driver", "SERVER", "NO_SSPS" but in your filter all of them is lowercase, this will cause your excepted lines absent in the archive. Please double confirm it.